Packs/FeedGCPWhitelist/Integrations/FeedGCPWhitelist/FeedGCPWhitelist_description.md

```diff
@@ -0,0 +1,3 @@
+## Hello World
+- This text is markdown
+- Here you should explain how to configure the instance in Demisto
```

Comment:
Update the description here to include information about what this integration does and it's purpose.